style.css

```diff
@@ -7,15 +7,15 @@
   width: 100vw;
```

Comment:
- [ ] Kindly remove the horizontal scroll on the page you can set the header width to a 100% this will fix the horizontal scrolling and make the page more responsive